modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/Constants.java

```diff
@@ -24,66 +23,73 @@
  */
 public class Constants {
 
-	public static final String PROVIDER_ID = "MobiCDMServer";
-	public static final String SERVER_SECRET = "dummy";
-	public static final String INITIAL_NONCE = "ZHVtbXk=";
-	public static final String DISENROLL_ALERT_DATA = "1226";
+    public static final String PROVIDER_ID = "MobiCDMServer";
+    public static final String SERVER_SECRET = "dummy";
+    public static final String INITIAL_NONCE = "ZHVtbXk=";
+    public static final String DISENROLL_ALERT_DATA = "1226";
+    public static final String INITIAL_ALERT_DATA = "1201";
+    public static final int EMPTY = 0;
 
-	public static final String SYNCML_ROOT_ELEMENT_NAME = "SyncML";
-	public static final String XMLNS_SYNCML = "SYNCML:SYNCML1.2";
-	public static final String UTF_8 = "UTF-8";
-	public static final String YES = "yes";
+    public static final String SYNCML_ROOT_ELEMENT_NAME = "SyncML";
+    public static final String XMLNS_SYNCML = "SYNCML:SYNCML1.2";
+    public static final String UTF_8 = "UTF-8";
+    public static final String MD5 = "MD5";
+    public static final String YES = "yes";
+    public static final String ERROR = "Error";
 
```

Comment:
It would be much cleaner if these constants are moved to a meaningful class (like SyncMLResponseCodes).